CryptoCoarseFundamentalUniverseSelectionFrameworkAlgorithm.cs

```diff
@@ -0,0 +1,87 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using QuantConnect.Brokerages;
+using QuantConnect.Data;
+using QuantConnect.Data.UniverseSelection;
+using QuantConnect.DataSource;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    /// <summary>
+    /// Example algorithm using the custom data type as a source of alpha
+    /// </summary>
+    public class CryptoCoarseFundamentalUniverseSelectionFrameworkAlgorithm : QCAlgorithm
+    {
+        /// <summary>
+        /// Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must be initialized.
+        /// </summary>
+        public override void Initialize()
+        {
+            SetStartDate(2020, 6, 1);
+            SetEndDate(2020, 6, 5);
+            SetCash(100000);
+
+            SetBrokerageModel(BrokerageName.GDAX, AccountType.Cash);
+
+            // Warm up the security with the last known price to avoid conversion error
+            SetSecurityInitializer(security => security.SetMarketPrice(GetLastKnownPrice(security)));
```

Comment:
Should probably be standard with the changes in https://github.com/QuantConnect/Lean.DataSource.CryptoCoarseFundamentalUniverse/pull/15/files#diff-4ff7ed3b7ffe8904928dd73ac42ace8f4c351103c47d45532f4cc4ceea9d468bR44 ?